### PR TITLE
Unit tests for calculateExpenses

### LINKED ISSUE #55: Test: CalculateExpenses helper function

### PR DESCRIPTION
**Pull request by mikejensen0**
resolves #55 

Made several unit tests for `calculateExpenses`. There are likely more cases to be tested, but I think these initial ones covers most cases.

Use `npm test` to run all tests.

Should give the following result:
image

Note that some edge cases not included are `max` and `min` value testing, but limitations to this should probably be implemented in the application altogether, as MobilePay does not support too small or too large transactions.

Lastly, support for import/export syntax has been added to `jest.config.js`, by adding babel to the transform, which is stated on Jest's website as a thing that should be done.

### CODE REVIEW

File: __tests__/unitTests/CalculateExpenses.test.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            { user: 'Bob', amount: 0 },
+            { user: 'Charlie', amount: 0 },
+
+        ];
+        const expectedTransactions: Transaction[] = [
+            { from: 'Bob', to: 'Alice', amount: 100/3 }, //33.33 repeating of course
+            { from: 'Charlie', to: 'Alice', amount: 100/3 },
+        ];
+        const result = calculateExpenses(expenses);
+        result.forEach((transaction, i) => expect(transaction.amount).toBeCloseTo(expectedTransactions[i].amount, 2));
```
Comment (UnluckyBird) [inline __tests__/unitTests/CalculateExpenses.test.ts:65]: Should calculate expenses not give transactions amounts with max 2 decimal points?

**Review by UnluckyBird — APPROVED** [review]

File: __tests__/unitTests/CalculateExpenses.test.ts (hunk 1 above)
Comment (mikejensen0) [reply __tests__/unitTests/CalculateExpenses.test.ts:65]: That could work as well. Currently, only 2 decimals are shown on the front-end, so I think it makes no difference.

**Review by andreasalstrup — APPROVED** [review]
LGTM .. but little question

File: __tests__/unitTests/CalculateExpenses.test.ts (hunk 1 above)
Comment (mikejensen0) [reply __tests__/unitTests/CalculateExpenses.test.ts:65]: Made a new PR, which should implement this issue as well:
https://github.com/andreasalstrup/CoShare/issues/71
